src/Microsoft.ML.TensorFlow/TensorFlow/TensorflowUtils.cs

```diff
@@ -75,27 +75,25 @@ internal static Schema GetModelSchema(IExceptionContext ectx, TFGraph graph, str
         /// of kind <see cref="OpType"/>, indicating the operation type of the node, and if that node has inputs in the graph,
         /// it contains metadata of kind <see cref="InputOps"/>, indicating the names of the input nodes.
         /// </summary>
-        /// <param name="ectx">An <see cref="IExceptionContext"/>.</param>
-        /// <param name="modelFile">The name of the file containing the TensorFlow model. Currently only frozen model
-        /// format is supported.</param>
-        public static Schema GetModelSchema(IExceptionContext ectx, string modelFile)
+        /// <param name="env">The environment to use.</param>
+        /// <param name="modelPath">Model to load.</param>
+        public static Schema GetModelSchema(IHostEnvironment env, string modelPath)
```

Comment:
>env [](start = 61, length = 3)

This is only used as an IExceptionContext, so instead of changing it here to IHostEnvironment, you can change LoadTensorFlowModel to take an IExceptionContext.